internal/vendors/accessor_test.go

```diff
@@ -6,15 +6,12 @@ import (
 	"errors"
 	"log"
 	"testing"
+	"time"
 
 	"github.com/DATA-DOG/go-sqlmock"
 	"github.com/onsi/gomega"
 )
 
-func Test_newPostgresVendorAccessor(t *testing.T) {
-	_ = newPostgresVendorAccessor(nil)
-}
-
```

Comment:
keep this, for coverage and ensuring the function doesn't panic

Reply:
Ahh okay, unintentionally deleted while i was resolving conflicts My bad ✌️